### PR TITLE
added functionality to navbar

### PR DESCRIPTION
**Pull request by joybiswas704**

### CODE REVIEW

File: src/components/Navbar/Navbar.js [hunk 1]
```diff
@@ -1,41 +1,103 @@
...
+            <Box p='4' onClick={() => history.push('/cart')}>
+                <IconButton
+                    variant='outline'
+                    colorScheme='teal'
+                    icon={<AiOutlineShoppingCart />}
+                />
+                <Badge
+                    variant='outline'
+                    bg='white'
+                    colorScheme='teal'
```
Comment (zealptl) [inline src/components/Navbar/Navbar.js:92]: use colors.colorScheme

File: src/components/Navbar/Navbar.js [hunk 2]
```diff
@@ -1,41 +1,103 @@
...
+                    />
+                </Box>
+            ) : (
+                ''
+            )}
+
+            <Box p='4' onClick={() => history.push('/cart')}>
+                <IconButton
+                    variant='outline'
+                    colorScheme='teal'
```
Comment (zealptl) [inline src/components/Navbar/Navbar.js:86]: use colors.colorScheme

File: src/components/Navbar/Navbar.js (hunk 1 above)
Comment (joybiswas704) [reply src/components/Navbar/Navbar.js]: done

File: src/components/Navbar/Navbar.js (hunk 2 above)
Comment (joybiswas704) [reply src/components/Navbar/Navbar.js]: done

**Review by zealptl — APPROVED** [review]

**Pull request merged**